contracts/levels/Fallback.sol

```diff
@@ -1,22 +1,35 @@
 pragma solidity ^0.5.0;
 
-import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
 import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
 
-contract Fallback is Ownable {
+contract Fallback {
 
   using SafeMath for uint256;
   mapping(address => uint) public contributions;
+  address payable _owner;
 
   constructor() public {
+    _owner = msg.sender;
     contributions[msg.sender] = 1000 * (1 ether);
   }
 
+   function owner() public view returns (address payable) {
```

Comment:
Instead of writing the manual `owner()` getter, it would be ok to just use a `public` variable directly. We don't do that in OpenZeppelin because we want to have internal variables, but that's not a concern here.

Reply:
Ok, thank you very much!